.github/CONTRIBUTING.md

```
@@ -127,20 +127,44 @@ This repository uses Git hooks to enforce commit message conventions and other c
+```mermaid
+sequenceDiagram
+    participant Dev as Developer
+    participant CMH as Commit Hook
+    participant PS as parse-title.sh
+    Dev->>CMH: Commit with message
+    CMH->>PS: Validate commit message title
+    PS-->>CMH: Return validation result
+    CMH-->>Dev: Commit allowed or error displayed
+```
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)